### ISSUE TITLE
postcss should be listed as a dep not a peer dep?

### ISSUE THREAD

**Issue by niftylettuce** [unlight/postcss-import-url#13]
a lot of users just use something like `gulp-postcss` and in turn this causes this error when installing this package:

```sh
warning " > postcss-import-url@3.0.1" has unmet peer dependency "postcss@^6.0.13".
```

I suggest to either put a version range for `postcss` in peer deps or strictly put a version of postcss as a dep

**Comment by unlight** [comment]
Postcss defined as [peer dependency](https://github.com/unlight/postcss-import-url/blob/38c5f212ac4245fa3a90092a7b882d0570790bfd/package.json#L20)
I'll set postcss as peerdep to `>=6 <7`

**Comment by niftylettuce** [comment]
Can you please accept my pull request?  This should be listed as a dependency.  From my research so far I think that almost all postcss plugins list is as a dependency.

**Comment by unlight** [comment]
> so far I think that almost all postcss plugins list is as a dependency.

Well, they do it wrong. Plugins must define host dependency as peer dependency.
[Proof link](https://nodejs.org/en/blog/npm/peer-dependencies/)

I've just published v3.0.3, if problem with warning still occurs, please provide your `package.json` content.

**Comment by niftylettuce** [comment]
I understand, however the community as a whole really doesn't seem to adhere to peerDependencies and the thought process there.
